app/routes.py

```diff
@@ -71,7 +71,11 @@ def download_image(request: DownloadImageRequest):
 
     image = Image.open(BytesIO(response.content))
     draw = ImageDraw.Draw(image)
-    font = ImageFont.load_default()  # You can specify a custom font here
+    
+    # Load a TrueType font
+    font_path = "app/static/fonts/Arial.ttf"
```

Comment:
Where do I get this from?

Reply:
You can include the TrueType font file (e.g., Arial.ttf) in the `app/static/fonts/` directory of the project. This way, the application can load the font from a known location. If you don't have the font file, you can download it from a reliable source or use any other TrueType font you prefer.